common/lib/capa/capa/responsetypes.py

```diff
@@ -343,9 +343,9 @@ def make_hint_div(self, hint_node, correct, student_answer, question_tag,
         # Tricky: label None means output defaults, while '' means output empty label
         if label is None:
             if correct:
-                label = _(u'Correct')
+                label = u'<span class="icon fa fa-check" aria-hidden="true"></span>Correct'
```

Comment:
There are a couple of issues here--

1. Correct and Incorrect are no longer getting translated
2. What used to be just plain text is now a combination of HTML and plain text. This will be difficult to get right for XSS, and is also ugly.

Can the correct/incorrect state be passed as a context variable so that the HTML can be directly in the template (with the icon type determined by the correct/incorrect state). Perhaps correct/incorrect is already being passed even.